source/features/unwrap-unnecessary-dropdowns.tsx

```diff
@@ -14,7 +14,8 @@ function replaceDropdownInPlace(dropdown: Element, form: Element): void {
 
 async function unwrapNotifications(): Promise<void | false> {
 	await elementReady('.js-check-all-container > :first-child'); // Ensure the entire dropdown has loaded
```

Comment:
This should be quick to fix the tests, even if not completely accurate. Replace `elementReady` with `observe('.js-check-all-container > :first-child')`